stan/math/rev/core/var.hpp

```diff
@@ -399,12 +301,13 @@ class var {
    * (a /= b) behaves exactly the same way as (a = a / b).  Note
    * that the result is an assignable lvalue.
    *
+   * @tparam Arith An arithmetic type
    * @param b The scalar to divide this variable by.
    * @return The result of dividing this variable by the specified
    * variable.
    */
-  template <typename Arith, require_arithmetic_t<Arith>...>
-  inline var& operator/=(Arith b);
+  template <typename Arith, require_arithmetic_t<Arith>* = nullptr>
+  inline var_value<T>& operator/=(Arith b);
```

Comment:
Similarly to the `is_convertible` stuff, why the template checks?

```
inline var_value<T>& operator/=(T b);
```

Will allow all the implicit conversions we want.

Reply:
Yes good catch!